docs/demo/general-form-elements.html

```diff
@@ -16,46 +16,46 @@ <h4 class="sub-heading">Input Types</h4>
             <div class="col-md-6">
                 <div class="form-group">
                     <label class="control-label">Text</label>
-                    <input type="text" class="form-control" value="Some text">
+                    <input id="sample-text" type="text" class="form-control" value="Some text">
```

Comment:
-1 on having a 13 character prefix for ID field.
Let's have a set of standard for naming.